packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -284,8 +284,13 @@ export function wrapErrorAndLog<T extends LoggingError>(
 	return newError;
 }
 
-function overwriteStack(error: IFluidErrorBase | LoggingError, stack: string): void {
-	// supposedly setting stack on an Error can throw.
+/**
+ * Attempts to overwrite the error's stack
+ *
+ * There have been reports of certain JS environments where overwriting stack will throw.
+ * If that happens, this adds the given stack as the telemetry property "stack2"
+ */
+export function overwriteStack(error: IFluidErrorBase | LoggingError, stack: string): void {
```

Comment:
Just adding the `@internal` tag. Will be relevant once we complete what @Josmithr has been working on, to not publish `@internal` APIs to our consumers while keeping them available in-repo.

```suggestion
/**
 * Attempts to overwrite the error's stack
 *
 * There have been reports of certain JS environments where overwriting stack will throw.
 * If that happens, this adds the given stack as the telemetry property "stack2"
 *
 * @internal
 */
export function overwriteStack(error: IFluidErrorBase | LoggingError, stack: string): void {
```